critiquebrainz/db/review.py

```diff
@@ -684,3 +684,28 @@ def reviewed_entities(*, entity_ids, entity_type):
         })
         reviewed_ids = [str(row[0]) for row in results.fetchall()]
     return reviewed_ids
+
+def get_top_contributors(limit=10):
+    query = sqlalchemy.text("""
+        SELECT review.user_id,
+            COUNT(*) as qtd,
+            "user".email,
+            "user".created as user_created,
+            "user".display_name,
+            "user".show_gravatar,
+            "user".musicbrainz_id,
+            "user".is_blocked
+        FROM review JOIN "user" 
+        ON review.user_id = "user".id
+        WHERE review.published_on > current_date - interval '7 days'
+        AND review.is_draft <> TRUE
+        GROUP BY review.user_id, "user".id
+        ORDER BY qtd DESC
+        LIMIT :limit
+    """)
```

Comment:
Please follow the generic indentation guidelines for SQL queries as per project preference. This applies to files, function, classes, dictionaries, and every other thing. 
For SQL queries, in particular, you can have a look at other SQL queries.
If you feel that your way is better than the current way, please discuss that with your mentor before moving on.

Reply:
We have some style guildelines documented here: https://github.com/metabrainz/guidelines/blob/master/Python.md

Specifically
>SQL keywords should be in upper-case and right aligned.

Reply:
Fixed. :smile_cat: